diagnostics/diag.go

```diff
@@ -98,7 +98,13 @@ func (di *DiagInfo) Marshal() []byte {
 }
 
 func (d *Diagnostics) getPeers() []peer.ID {
-	return d.host.Network().Peers()
```

Comment:
@whyrusleeping it is supposed to :/ we can fix it in there. too, whatever you prefer